PublishingProcess.md

```diff
@@ -0,0 +1,48 @@
+# The Publishing Process for Atoms
+This is the publishing process for Atom Workshop and recommended best practices for other Tools that create Atoms.
+
+This project uses the [atom maker](https://github.com/guardian/atom-maker/) for publishing, reindexing and saving atoms in a
+DynamoDB. As a result there is the concept of Drafts (atoms where all fields are optional) and Atoms where whether or not a
+field is optional depends on the [thrift definition](https://github.com/guardian/content-atom).
+
+There are 3 stages an atom can be in:
+### Draft
+Not all of the fields required to publish an atom have been supplied. This stage allows atoms to be saved as a work in progress rather than
+having to fill in all of fields of the atom that are required according to the thrift definition for atoms.
+### Preview
+At this stage an atom is ready to be published but has not yet been published. The purpose is to allow atoms to be previewed in their
+complete state by editorial before they go live.
+### Live
+At this stage an atom has been published. It can be seen on the site in any content it has been embedded in.
+
+## Publishing flow
+1. Create atom
+2. The atom is saved in Draft form in the DraftsDynamoDataStore while the user has not filled out all the fields required according to the thrift
+definition. This enables the user to save their work at any point. At this point in the flow the atom is not in [Content API](http://open-platform.theguardian.com/documentation/).
+3. Every time an atom is saved to the DraftsDynamoDataStore atom workshop also checks if all the required fields have been
+supplied. If they have the atom is put in PreviewDynamoDataStore and put into preview CAPI. This step is not visible to the user.
+4. The atom is now in a publishable state. The 'publish' button in the UI becomes active.
+5. On selecting 'Publish' the atom is put in the PublishedDynamoDataStore and in live CAPI. Once published the atom can be embedded in
+composer content.
+
+## Editing Rules
+When editing an atom what is saved where and when?
+Note: When referring to automatic save, the save is automatic from the user's perspective. The only manual step for the user is publishing.
+### Scenario 1: An atom is in drafts only
+Changes are automatically saved to DraftsDynamoDataStore. If the changes have made the atom valid according to the thrift definition,
+i.e. all required fields are provided, the atom is also saved to PreviewDynamoDataStore and preview CAPI is updated
+to include the atom. The user does not have to do anything to transition the atom from Drafts to Preview. It is possible to publish the atom.
```

Comment:
I think it might be confusing to end a section of draft atoms with this sentence, maybe just add `once the atom is in preview` or something similar.